src/stylesheets/components/_nav.scss

```diff
@@ -41,15 +44,18 @@
 
   ul {
     align-items: center;
+    background-color: $florence-red;
+    border-radius: 999px;
+    box-shadow: inset 0 -16px 0 rgba(0, 0, 0, 0.1);
     display: flex;
     flex-direction: row;
     font-size: 0.65em;
     justify-content: space-around;
     list-style: none;
     margin: auto;
     text-transform: uppercase;
-    width: 100%;
-    
+    width: 500px;
```

Comment:
maybe i can't see if this is within a breakpoint, but i think you should leave the `width: 100%` and set `max-width: 500px`. this way, if the screen is under 500px this element will be as will be as wide as its parent container, but over 500px it will lock to this dimension. i have a feeling this will create overflow and scroll horizontally off the page at a mobile breakpoint as is.

Reply:
ah ha, that makes sense